torch_ort_inference/torch_ort/ortinferencemodule/_utils_infer.py

```diff
@@ -0,0 +1,133 @@
+# -------------------------------------------------------------------------
+# Copyright (C) 2022 Intel Corporation
+# Licensed under the MIT License
+# --------------------------------------------------------------------------
+
+import functools
+from collections import abc
+from onnxruntime.training.ortmodule import _io
+
+def get_device_from_module(module):
```

Comment:
This is fine for now. We should think of a way to reuse ortmodule utilities in the future, so we don't have to maintain similar code in multiple places.